content/authorization.md

```diff
@@ -382,6 +383,20 @@ where the audience is provided:
 &aud=https%3A%2F%2Ffhir-ehr.sandboxcerner.com%2Fdstu2%2F0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca%2F
 </pre>
 
+#### Redirect URI ####
+
+When performaing an authorization grant request the application may provide a redirect uri. 
+This uri should match the redirect uri setup during the registration phase of the SMART Application.
+
+If your application has multiple redirect uri's registered with cerner then a redirect uri is expected to be passed
+in during the authorization grant request. If no redirect uri is specified the default redirect uri 
+registered for the application will be used.
+
+Details around supplying a redirect uri are specified in the [SMART Specification](http://www.hl7.org/fhir/smart-app-launch/#step-1-app-asks-for-authorization)
+
```

Comment:
Please add an inline example code block to show the construction of the `redirect_uri` segment.

Reply:
Added an inline example block and a complete example link : https://github.com/cerner/fhir.cerner.com/pull/370/commits/f833a4f6ab9a21f7028085efb35ab19ef0a578b7